src/zectl_create.c

```diff
@@ -14,7 +14,7 @@
 libze_error
 ze_create(libze_handle *lzeh, int argc, char **argv) {
 
-    libze_bootloader bootloader;
+    libze_bootloader bootloader = { 0 };
```

Comment:
It appears this was actually left in accidentally, the bootloader is initialized already somewhere else, so this actually does nothing.

Could you remove all traces of the bootloader from this file? I tested without a bootloader after removing the configuration from this file and didn't have any issues.

```diff
diff --git a/src/zectl_create.c b/src/zectl_create.c
index 74029c7..951657d 100644
--- a/src/zectl_create.c
+++ b/src/zectl_create.c
@@ -14,12 +14,9 @@
 libze_error
 ze_create(libze_handle *lzeh, int argc, char **argv) {
 
-    libze_bootloader bootloader;
-
     libze_create_options be_clone = {.existing = B_FALSE, .recursive = B_FALSE};
 
     libze_error ret = LIBZE_ERROR_SUCCESS;
-    bootloader.set = B_FALSE;
 
     char *be_existing = NULL;
 
@@ -55,11 +52,6 @@ ze_create(libze_handle *lzeh, int argc, char **argv) {
         return LIBZE_ERROR_MAXPATHLEN;
     }
 
-    if ((ret = libze_bootloader_init(lzeh, &bootloader, ZE_PROP_NAMESPACE)) !=
-        LIBZE_ERROR_SUCCESS) {
-        goto err;
-    }
-
     if (be_clone.existing) {
         if (strlcpy(be_clone.be_source, be_existing, ZFS_MAX_DATASET_NAME_LEN) >=
             ZFS_MAX_DATASET_NAME_LEN) {
@@ -71,6 +63,5 @@ ze_create(libze_handle *lzeh, int argc, char **argv) {
     ret = libze_create(lzeh, &be_clone);
 
 err:
-    libze_bootloader_fini(&bootloader);
     return ret;
 }
```